great_expectations/datasource/fluent/pandas_datasource.pyi

```diff
@@ -143,12 +143,14 @@ class PandasDatasource(_PandasDatasource):
     type: Literal["pandas"]
     assets: Dict[str, _PandasDataAsset]
     def test_connection(self, test_assets: bool = ...) -> None: ...
-    def _get_validator(self, asset: _PandasDataAssetT) -> Validator: ...
+    @staticmethod
+    def _validate_asset_name(self, asset_name: Optional[str] = ...) -> str: ...
+    def _get_validator(self, asset: _PandasDataAsset) -> Validator: ...
```

Comment:
Why do these need to be part of the stub?
Are they being called from outside the class?

Reply:
@Kilo59 I forgot to follow up with you about this. I think they can actually be removed from the stubs (I removed them). It just results in the IDE complaining about "Unresolved attribute", but I think the stubs should be reserved for public methods.